### PR TITLE
More clippy fixes.

### PR DESCRIPTION
**Pull request by canndrew**
This change is [Reviewable](https://reviewable.io/reviews/maidsafe/safe_core/277)

### CODE REVIEW

**Comment by maidsafe-highfive** [comment]
r? @ustulation

(maidsafe_highfive has picked a reviewer for you, use r? to override)

File: examples/client_stress_test.rs [hunk 1]
```diff
@@ -113,13 +113,13 @@ fn main() {
     let pin_range = Range::new(0u16, 9999);
     let pin = pin_range.ind_sample(&mut rng).to_string();
 
-    let mut client = if !args.flag_get_only {
+    let mut client = if args.flag_get_only {
+        unwrap_result!(Client::log_in(keyword.clone(), pin.clone(), password.clone()))
```
Comment (dirvine) [inline examples/client_stress_test.rs]: Have these not been agreed to change to just the unwrap crate now?

Comment (canndrew) [inline examples/client_stress_test.rs]: Yes, I can go through and make change them all in the next commit.

**Comment by ustulation** [comment]
Reviewed 22 of 22 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion, some commit checks failed.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/maidsafe/safe_core/277)*

File: examples/client_stress_test.rs (hunk 1 above)
Comment (ustulation) [inline examples/client_stress_test.rs]: @dirvine you will have to acknowledge or something i think for Reviewable to green out below. I tried to mark it `Satisfied` but i think it looks for your acknowledgement.

**Pull request merged**